libs/s25main/network/GameClient.cpp

```diff
@@ -1759,7 +1759,7 @@ unsigned GameClient::GetTournamentModeDuration() const
             < rttr::enum_cast(GameObjective::Tournament1) + NUM_TOURNAMENT_MODES)
     {
         const auto turnamentMode = rttr::enum_cast(game->ggs_.objective) - rttr::enum_cast(GameObjective::Tournament1);
-        return minutes(TOURNAMENT_MODES_DURATION[turnamentMode]) / framesinfo.gf_length;
+        return minutes(TOURNAMENT_MODES_DURATION[turnamentMode]) / milliseconds(SPEED_GF_LENGTHS[referenceSpeed]);
```

Comment:
MSVC needs a `static_cast<unsigned>` here (shame that a duration is signed, but whatever)